src/js/components/form/FieldAutofocus.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import DOMUtils from "#SRC/js/utils/DOMUtils";
+
+let lockFieldAutofocus = false;
```

Comment:
It's unclear to me what impact this field is having. I can't reproduce a scenario where it changes the behavior. In what scenario is this needed?